src/modules/saveToDB.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
```

Comment:
- [ ] As mentioned before, kindly keep in mind that disabling `eslint` in your document should be avoided because `eslint` is a valuable tool that helps to identify errors and enhance the overall quality of your code.